devcycle_python_sdk/models/feature.py

```diff
@@ -141,29 +150,80 @@ def type(self, type):
 
     @property
     def variation(self):
-        """Gets the variation of this Feature.  # noqa: E501
+        """Gets the variation ID of this Feature.  # noqa: E501
 
-        Bucketed feature variation  # noqa: E501
+        Bucketed feature variation ID # noqa: E501
 
-        :return: The variation of this Feature.  # noqa: E501
+        :return: The variation ID of this Feature.  # noqa: E501
         :rtype: str
         """
         return self._variation
 
     @variation.setter
     def variation(self, variation):
-        """Sets the variation of this Feature.
+        """Sets the variation ID of this Feature.
 
-        Bucketed feature variation  # noqa: E501
+        Bucketed feature variation ID # noqa: E501
 
-        :param variation: The variation of this Feature.  # noqa: E501
+        :param variation: The variation ID of this Feature.  # noqa: E501
         :type: str
         """
         if variation is None:
             raise ValueError("Invalid value for `variation`, must not be `None`")  # noqa: E501
 
         self._variation = variation
 
+    @property
```

Comment:
Not sure if i should actually be writing these in manually. should I run something with swagger to generate them instead?

Reply:
I think youre fine to do this? This sdk was generated from swagger, rather than the other way around. Im not 100% how if you can autopopulate the swagger docs, I havent done it before